lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -3402,6 +3402,26 @@ class SPIRVJointMatrixINTELWorkItemInst : public SPIRVJointMatrixINTELInstBase {
 _SPIRV_OP(JointMatrixGetElementCoord, true, 5)
 #undef _SPIRV_OP
 
+class SPIRVCooperativeMatrixInvocationInstructionsINTELInstBase
+    : public SPIRVInstTemplateBase {
+protected:
+  std::optional<ExtensionID> getRequiredExtension() const override {
+    return ExtensionID::SPV_INTEL_joint_matrix;
+  }
+  SPIRVCapVec getRequiredCapability() const override {
+    return getVec(
+        internal::CapabilityCooperativeMatrixInvocationInstructionsINTEL);
```

Comment:
we also need `CapabilityCooperativeMatrixKHR`, right?

Reply:
The instruction has an argument of OpTypeCooperativeMatrixKHR, so it looks like yes.
@MrSidims can you please confirm if CapabilityCooperativeMatrixKHR must be also returned by getRequiredCapability() along with internal::CapabilityCooperativeMatrixInvocationInstructionsINTEL?

Reply:
This capability will be added when we translate Cooperative Matrix Type. Please also note, that while the specification says, that we rely on cooperative matrix extension, it doesn't say, that one capability implicitly declares another :)

Reply:
Please ignore my replay. I'm completely wrong.

Reply:
so we still should add the capability? or is it enabled implicitly elsewhere?

Reply:
Slava will update the PR, SPIRV/libSPIRV/SPIRVEnum.h should be updated